src/plugins/condition/ConditionManager.js

```diff
@@ -142,7 +143,7 @@ export default class ConditionManager extends EventEmitter {
     }
 
     createAndSaveCondition(index, conditionConfiguration) {
-        let newCondition = this.createCondition(conditionConfiguration);
+        const newCondition = _.cloneDeep(this.createCondition(conditionConfiguration));
```

Comment:
The `cloneCondition` method exists in ConditionManager. You should make this change there.
Recommend something like: 
`cloneCondition(conditionConfiguration, index) {
        this.createAndSaveCondition(index, Object.assign({}, conditionConfiguration));
}`

Reply:
I tried Object.assign first, but it apparently doesn't work because it's not a deep copy. Pretty good discussion here: https://stackoverflow.com/questions/38416020/deep-copy-in-es6-using-the-spread-syntax